src/components/UsersList/UsersList.tsx

```diff
@@ -329,12 +331,14 @@ export function UsersList({
               </TableCell>
 
               <TableCell className="text-center mobile-lg:text-right">
-                <UsersActionsRemoveFromOrg
-                  user={user}
-                  roles={roles[user.guid]}
-                  removeUserCallback={removeUserCallback}
-                  closeOnSuccess={true}
-                />
+                {allowChanges && (
```

Comment:
Should this wrap `<TableCell />` instead so an empty column does not display if folks don't have edit permissions?